environment_set_up/QuickStart-Cloud.md

```diff
@@ -0,0 +1,11 @@
+# Quick-Start for Cloud
+
+This suite of tutorials includes background material, notes, and introductory
+tutorials to lay groundwork for running HyTEST notbooks on an existing
+Jupter Hub service in the cloud.
+
+## 1) One
+
+## 2) Two
+
+## 3) Three
```

Comment:
We can delete these later...just copying in what we have from Rich's md file so we can delete it if this replaces it.
```suggestion
## 1) Obtain access to pangeo.chs.usgs.gov

## 2) Clone a repository containing example notebooks

## 3) Run notebooks in JupyterLab
```

Reply:
Done